tests/cpu/test_kalman_fitter.cpp

```diff
@@ -0,0 +1,193 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+// Project include(s).
+#include "tests/seed_generator.hpp"
+#include "traccc/edm/track_state.hpp"
+#include "traccc/fitting/fitting_algorithm.hpp"
+#include "traccc/resolution/fitting_performance_writer.hpp"
+
+// detray include(s).
+#include "detray/detectors/create_telescope_detector.hpp"
+#include "detray/propagator/actors/aborters.hpp"
+#include "detray/propagator/actors/parameter_resetter.hpp"
+#include "detray/propagator/actors/parameter_transporter.hpp"
+#include "detray/propagator/actors/pointwise_material_interactor.hpp"
+#include "detray/propagator/propagator.hpp"
+#include "detray/simulation/simulator.hpp"
+#include "detray/simulation/track_generators.hpp"
+
+// VecMem include(s).
+#include <vecmem/memory/host_memory_resource.hpp>
+
+// GTest include(s).
+#include <gtest/gtest.h>
+
+// ROOT include(s).
+#include <TF1.h>
+
+// System include(s).
+#include <filesystem>
```

Comment:
I don't think this include is needed here.

Reply:
Fixed